caravel/config.py

```diff
@@ -121,6 +121,38 @@
 CACHE_DEFAULT_TIMEOUT = None
 CACHE_CONFIG = {'CACHE_TYPE': 'null'}
 
+
+# ---------------------------------------------------
+# Enable different visualizations: True or False (refer viz.py)
+# ---------------------------------------------------
+
+VIZ_TYPES = {
```

Comment:
I'd rather go with a blacklist so that we don't have to touch this everytime we add a new viz, probably should just be a tuple of viz_types